lib/transaction/transaction.go

```diff
@@ -175,6 +178,7 @@ func (tx Transaction) String() string {
 }
 
 func (tx *Transaction) Sign(kp keypair.KP, networkID []byte) {
+	tx.B.Source = kp.Address()
```

Comment:
What is it?

Reply:
@kfangw I found the missing part of this. As we know, `Sign()` must be done by `Source` account and the existing code set the `Source` manually.